protocol/x/clob/keeper/stateful_order_state.go

```diff
@@ -518,8 +518,7 @@ func (k Keeper) getUntriggeredConditionalOrdersIterator(ctx sdk.Context) sdk.Ite
 	return sdk.KVStorePrefixIterator(store, []byte{})
 }
 
-// GetStatefulOrderCount gets a count of how many stateful orders are written to state for a subaccount. This does not
-// include any untriggered conditional orders.
+// GetStatefulOrderCount gets a count of how many stateful orders are written to state for a subaccount.
```

Comment:
Can we update [this comment](https://github.com/dydxprotocol/v4-chain/blob/e48f473f1771b6c9e8d6f4862cc9a7131ea5b701/protocol/x/clob/keeper/stateful_order_state.go#L538) too?